Makefile

```diff
@@ -120,20 +120,17 @@ vet:
 
 PHONY+= dist-release
 dist-release:
```

Comment:
This is pretty brittle, obviously (e.g. will break if we go back to multiple binaries, etc.) but I went for explicit and simple for the time being. It's easy enough to change later as required.